PolyEngine/Engine/Src/LightSourceComponent.cpp

```diff
@@ -2,17 +2,17 @@
 
 #include "LightSourceComponent.hpp"
 
-Poly::AmbientLightWorldComponent::AmbientLightWorldComponent(const Color& color, float intensity)
+Poly::AmbientLightWorldComponent::AmbientLightWorldComponent(const Color& color = Color(1.0f, 1.0f, 1.0f), float intensity = 1.0f)
```

Comment:
`Color(1.0f, 1.0f, 1.0f)` is used very often, lets provide static fields for each common color, like: `Color::RED`, `Color::BLACK`, etc. Use these fields for initialization.
Similar thing is done in Vector class.

Reply:
Done